src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/pathverification/PathVerificationService.java

```diff
@@ -601,7 +604,7 @@ private void handleDiscoveryPacket(OfInput input, IOFSwitch destSwitch, Discover
                 source.getPortNo(),
                 destination.getSwitchId(),
                 destination.getPortNo(),
-                latency,
+                scaleLatencyMsToNs(latencyMs),  // TODO(surabujin): do we really need fake ns accuracy?
```

Comment:
yes, we need in because this value could be used in
1. Isl latency database field
2. opentsdb `rtt` metric
both these metrics are in nano seconds

Reply:
So we should make this conversion right before storing it in the mentioned storage. At least the fake ns accuracy conversion reason will be more obvious.